src/schnetpack/interfaces/batchwise_optimizer.py

```diff
@@ -0,0 +1,109 @@
+import torch
+import logging
+from tqdm import tqdm
+from schnetpack import properties
+
+
+__all__ = ["TorchLBFGS"]
+
+
+class TorchLBFGS(torch.optim.LBFGS):
```

Comment:
perhaps rename it to make clear it is for geometry optimization? otherwise it is not directly clear why we modify the torch implementation. also point out wht the changes are in the docstring